src/core/Map/Layer/index.tsx

```diff
@@ -21,6 +21,7 @@ export type FeatureComponentProps = {
   onFeatureRequest?: (range: DataRange) => void;
   onFeatureFetch?: (features: Feature[]) => void;
   onFeatureDelete?: (features: string[]) => void;
+  evalFeature?: (layer: LayerSimple, feature: Feature) => Feature | undefined;
```

Comment:
```suggestion
  evalFeature?: (layer: LayerSimple, feature: Feature) => ComputedFeature | undefined;
```

Reply:
I fixed this comment by below commits.
[Update src/core/Map/Layer/index.tsx](https://github.com/reearth/reearth-web/pull/388/commits/803e2ba26e7c5a4ba4042b64f0e1d7a326c34a0c)
[fix: import error](https://github.com/reearth/reearth-web/pull/388/commits/2455a951a45eccf64f20088c8888c3840db647b1)